### PR TITLE
Reduce dependabot noise

### PR DESCRIPTION
**Pull request by JAVGan**
Set it to run monthly :)

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -3,6 +3,6 @@ updates:
 - package-ecosystem: pip
   directory: /
   schedule:
-    interval: daily
+    interval: monthly
```
Comment (lslebodn) [inline .github/dependabot.yml:6]: Looks like even weekly is supported
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval

Have you checked "[Grouping Dependabot updates into one pull request](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/customizing-dependency-updates#grouping-dependabot-updates-into-one-pull-request)"

There seems to be similar for security updates
https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/about-dependabot-security-updates#about-grouped-security-updates

Comment (JAVGan) [reply .github/dependabot.yml:6]: Thanks!

I'll see if we can group some deps and then set it as weekly

Comment (JAVGan) [reply .github/dependabot.yml:6]: Agreed!

I still need time to properly configure it in the way it works weekly with groups and everything.

I'll open similar PRs for other 2 projects, please ack here and there as well once opened, let's have a pacefull weekend 😄

Comment (JAVGan) [reply .github/dependabot.yml:6]: There:

- https://github.com/release-engineering/pubtools-marketplacesvm/pull/33
- https://github.com/release-engineering/cloudpub/pull/32

**Comment by JAVGan** [comment]
@lslebodn or @jajreidy mind acking to set this the same of other repos?

Then later on we'll check how to properly group deps

**Review by lslebodn — APPROVED** [review]
LGTM

**Pull request merged**